src/components/orderDetails.js

```diff
@@ -10,12 +10,56 @@ function prettyDate(rawDate) {
     return humanDate;
   }
 
+  class ExchangeTotal extends React.Component {
+    constructor(props) {
+        super(props)
+    }
+    render () {
+        return (
+            <div>
+                <p>{this.props.count}</p>
+            </div>
+        )
+    }
+}
+
+
+class RefundTotal extends React.Component {
+  constructor(props) {
+      super(props)
+  }
+  calRefundTotal(){
+    const qty = this.props.qty;
+    const price = 100;
+    const result = price * qty;
+    return result;
+  }
+  render () {
+      return ( 
+          <div>
+          ${this.calRefundTotal()}
+          </div>
+      )
+  }
+}
+
 class OrderDetails extends React.Component {
     
     constructor(props) {
         super(props)
+        this.state = {
+            quantityToReturn: 0,
+            quantityToExchange: 0,
```

Comment:
If the `<OrderDetails />` component contains all of the items within the order, it should probably contain a `returnDetails` object in its state. Storing the quantity to return and the quantity to exchange is going to make it hard to figure out how many of which specific item will be returned/exchanged.

The `returnDetails` object will probably end up looking like something similar to the return details object in [the documentation](https://docs.google.com/document/d/1xBk5xdV34KGzlhiBqPHqllHVhu7BIynAZUYEQXGgPyQ/edit#heading=h.7r5zg59skhcw), for example:

```json
{
    "return_items": [
        {
            "level_code": "ABC_1234",
            "quantity": 1,
            "reason": "FAULTY",
            "photos": [
                "75801619-794b-477c-b495-2f268dfbcbfd",
                "79bda7e6-d829-4dfd-ae58-4e12a2856eaa",
                "a810570f-7497-4b15-8fc0-95e20b77cede"
            ],
            "comments": "The sole is missing from the shoe"
        }
    ]
}
```

This comment is not for you to act on now (this merge can go through) -- this is for future reference.